demos/jans-tarp/src/options/addCedarlingConfig.tsx

```diff
@@ -52,6 +67,16 @@ export default function AddCedarlingConfig({ isOpen, handleDialog, newData }) {
     setOpen(true);
   };
 
+  const copyToClipboard = () => {
+    try {
+      const jsonString = JSON.stringify(bootstrap, null, 2); // pretty print
+      navigator.clipboard.writeText(jsonString);
+      console.log("JSON copied to clipboard!" + jsonString);
+    } catch (error) {
+      console.error("Copy failed:", error);
```

Comment:
I suggest using toast or alert is more user friendly instead of console log

Reply:
done